src/components/Person/Person.jsx

```diff
@@ -1 +1,39 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+import './Person.scss';
+
+export const Person = ({ person }) => {
+  const {
+    name,
+    age,
+    sex,
+    isMarried,
+    partnerName,
+  } = person;
+
+  const spouse = isMarried
+    && sex === 'm'
```

Comment:
a little overkill :)
```suggestion
  const spouse = sex === 'm'
```